solutions/swb-reference/src/SWBStack.ts

```diff
@@ -844,7 +844,7 @@ export class SWBStack extends Stack {
   ): WorkbenchCognito {
     const props: WorkbenchCognitoProps = {
       domainPrefix: domainPrefix,
-      websiteUrl: websiteUrl,
+      websiteUrls: [websiteUrl],
```

Comment:
@nguyen102 I'm not sure hoe SWB handles multiple allowed origins, but each origin will need to be included in this list.